docs/src/main/tut/iso.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title:  "Iso"
+section: "optics"
+scaladoc: "http://julien-truffaut.github.io/Monocle/api/#monocle.PLens"
+source: "https://github.com/julien-truffaut/Monocle/blob/master/example/src/main/scala/monocle/example/LensExample.scala"
+pageSource: "https://raw.githubusercontent.com/julien-truffaut/Monocle/master/docs/src/main/tut/iso.md"
+---
+
+# Iso
+
+An `Iso` is an Optic which converts elements of type `A` into elements of type
+`B` without loss.
+
+Consider these two case classes:
+
+```tut:silent
+case class Person(name: String, firstName : String)
+case class Pers(n: String, f: String)
+```
+
+In order to create an `Iso` between `Person` and `Pers` we need to supply two total functions
+
+* get : Person => Pers
+* reverseGet : Pers => Person
+
+```tut:silent
+import monocle._
+val personPrism = Iso[Person, Pers]((p: Person) => Pers(p.name, p.firstName))((p: Pers) => Person(p.n, p.f))
+```
+
+and thereby create a lossless conversion between these two types. Other usages include for example the transformation of different types of physical units.
+
+It is important to understand that the transformation between type `A` and `B` works for any type `A` and `B` and that the functions `get` and `reverseGet` are true inverses to each other.
```

Comment:
:+1: to mention that `get` and `reverseGet` are inverse of each other, we could also give the law that comes from this. However, I find the first sentence ("It is important") a bit confusing.